### ISSUE TITLE
Vulnerability in property display table

### ISSUE THREAD

**Issue by EricMedina024** [jneale/Xplore#48]
The property disable table which displays the name, type and value of each property of the last returned object is vulnerable to XSS in the name column. You can test this by running the following script which will pop an alert:
```
var x = {'<img onerror="javascript:alert(1)" src="fake.local"/>': 'xss'};
x
```
The result of running this can be seen below:
![image](https://user-images.githubusercontent.com/53099692/119681724-f0447980-be07-11eb-8668-4d51ac60cdd7.png)

**Comment by jneale** [comment]
Thanks.

**Comment by jneale** [comment]
Fixed in 4.9
